domains/shad.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Maverick00001",
+        "email": "saksham.access@yahoo.com"
+    },
+    "record": {
+        "A": ["185.199.108.153"]
```

Comment:
If you want to use github pages, it requires 4 IPs
- `185.199.108.153`
- `185.199.109.153`
- `185.199.110.153`
- `185.199.111.153`